src/System.Composition.AttributedModel/src/System/Composition/Convention/AttributedModelProvider.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
```

Comment:
@stephentoub It doesn't look like removing unnecessary using statements is explicitly part of the [Coding style guidelines](https://github.com/dotnet/corefx/wiki/Coding-style).  Is this something we want to add?